<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarehouseEdit</class>
 <widget class="QMainWindow" name="WarehouseEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>465</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>610</width>
    <height>465</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>610</width>
    <height>465</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>安全库存:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>出库时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label4">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>备件号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label3">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>140</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>序号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>入库时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>存放柜号:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CabinetCombo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
	font: 18pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="SpareEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>输入备件号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 15px;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
	color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>确认修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReturnButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 15px;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
	color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OutTime">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EnterTime">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QComboBox" name="IDCombo">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
	font: 18pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LogoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>53</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/images/logo200x53.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="BackGround">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>601</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border-style: outset;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="SaveNum">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 221, 255);
border: none;
border-radius: 2px;
color: rgb(0,0,0);
font: 16pt &quot;微软雅黑&quot;;
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QLabel" name="Label6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>期初库存:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="FirstNum">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 221, 255);
border: none;
border-radius: 2px;
color: rgb(0,0,0);
font: 16pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label7">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>总库存量:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="TotalNum">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 221, 255);
border: none;
border-radius: 2px;
color: rgb(0,0,0);
font: 16pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLabel" name="Label9">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>累计入库:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="EnterNum">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 221, 255);
border: none;
border-radius: 2px;
color: rgb(0,0,0);
font: 16pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="OutNum">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 221, 255);
border: none;
border-radius: 2px;
color: rgb(0,0,0);
font: 16pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label11">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>累计出库:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MoldButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 15px;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
	color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>模具修改</string>
    </property>
    <property name="icon">
     <iconset resource="../resources.qrc">
      <normaloff>:/images/maintenance.png</normaloff>:/images/maintenance.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模具编号:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="MoldCombo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
	font: 18pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="PartLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>冲压部门</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OtherEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="placeholderText">
     <string>输入备注信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>备注信息:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="PriceNum">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 221, 255);
border: none;
border-radius: 2px;
color: rgb(0,0,0);
font: 16pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label13">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>备件价格:</string>
    </property>
   </widget>
   <zorder>BackGround</zorder>
   <zorder>Label5</zorder>
   <zorder>Label10</zorder>
   <zorder>Label4</zorder>
   <zorder>Label3</zorder>
   <zorder>Label8</zorder>
   <zorder>Label12</zorder>
   <zorder>CabinetCombo</zorder>
   <zorder>SpareEdit</zorder>
   <zorder>EditButton</zorder>
   <zorder>ReturnButton</zorder>
   <zorder>OutTime</zorder>
   <zorder>EnterTime</zorder>
   <zorder>IDCombo</zorder>
   <zorder>LogoLabel</zorder>
   <zorder>SaveNum</zorder>
   <zorder>Label6</zorder>
   <zorder>FirstNum</zorder>
   <zorder>Label7</zorder>
   <zorder>TotalNum</zorder>
   <zorder>Label9</zorder>
   <zorder>EnterNum</zorder>
   <zorder>OutNum</zorder>
   <zorder>Label11</zorder>
   <zorder>MoldButton</zorder>
   <zorder>Label1</zorder>
   <zorder>MoldCombo</zorder>
   <zorder>PartLabel</zorder>
   <zorder>OtherEdit</zorder>
   <zorder>Label2</zorder>
   <zorder>PriceNum</zorder>
   <zorder>Label13</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>MoldCombo</tabstop>
  <tabstop>OtherEdit</tabstop>
  <tabstop>IDCombo</tabstop>
  <tabstop>SpareEdit</tabstop>
  <tabstop>TotalNum</tabstop>
  <tabstop>SaveNum</tabstop>
  <tabstop>FirstNum</tabstop>
  <tabstop>EnterNum</tabstop>
  <tabstop>EnterTime</tabstop>
  <tabstop>OutNum</tabstop>
  <tabstop>OutTime</tabstop>
  <tabstop>PriceNum</tabstop>
  <tabstop>CabinetCombo</tabstop>
  <tabstop>EditButton</tabstop>
  <tabstop>ReturnButton</tabstop>
  <tabstop>MoldButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
